run

```diff
@@ -678,6 +678,9 @@ build() {
   run_command cmake --build ${build_path} -j
   ret=$?
   check_exit_code $ret "Error building application."
+  run_command cmake --install ${build_path}
+  ret=$?
+  check_exit_code $ret "Error installing application."
```

Comment:
Could you please add an `--install` CLI argument to opt in to this block? Status quo is building without packaging, having installation as the default will likely lead to redundant library copies

Reply:
Thanks. I have added a `--install` argument to the `build` command.